autoendpoint/src/error.rs

```diff
@@ -203,21 +212,32 @@ impl ApiErrorKind {
 
     /// Don't report all errors to sentry
     pub fn is_sentry_event(&self) -> bool {
-        !matches!(
-            self,
-            // Ignore common webpush errors
-            ApiErrorKind::NoTTL | ApiErrorKind::InvalidEncryption(_) |
-            // Ignore common VAPID erros
-            ApiErrorKind::VapidError(_)
-            | ApiErrorKind::Jwt(_)
-            | ApiErrorKind::TokenHashValidation(_)
-            | ApiErrorKind::InvalidAuthentication
-            | ApiErrorKind::InvalidLocalAuth(_) |
-            // Ignore missing or invalid user errors
-            ApiErrorKind::NoUser | ApiErrorKind::NoSubscription |
-            // Ignore overflow errors
-            ApiErrorKind::Router(RouterError::TooMuchData(_)),
-        )
+        match self {
+            // ignore selected validation errors.
+            ApiErrorKind::Validation(msg) => !msg.to_string().contains(TOPIC_LENGTH_ERR),
```

Comment:
It looks like we only have 3 distinct validators/`ValidationErrors` currently -- I see at least one of the others (Topic must be URL and Filename safe Base64 alphabet) also showing up a lot in [this event](https://mozilla.sentry.io/issues/3925602807/events/?project=6326250&referrer=issue-stream&statsPeriod=14d&stream_index=0) -- so maybe just ignore them all for now?

If we care more about these in the future we could always improve the metric labels to be more specific than just `"validation"` kind of like syncstorage does.